docs/root/intro/arch_overview/upstream/load_balancing/slow_start.rst

```diff
@@ -0,0 +1,66 @@
+.. _arch_overview_load_balancing_slow_start:
+
+Slow start mode
+===============
+
+Slow start mode is a configuration setting in Envoy to progressively increase amount of traffic for newly spawned service instances.
+With no slow start enabled Envoy would send a proportional amount of traffic to new instances.
+This could be undesirable for services that require warm up time to serve full production load and could result in request timeouts, loss of data and deteriorated user experience.
+
+Slow start mode is a weight-adjustment mechanism in number of load balancer types and can be configured per cluster basis. 
+Currently, slow start is supported in Round Robin and Least Request load balancer types.
+
+Users can specify a slow start window parameter (in seconds), so that if host’s “cluster membership duration" (amount of time since it has joined the cluster) is within the configured window, it enters slow start mode, given that the host satisfies endpoint warming policy. 
+Whenever a slow start window duration elapses, it exits slow start mode and gets regular amount of traffic acccording to load balanacing algorithm.
+Host could also exit slow start mode in case it leaves the cluster.
+
+To reiterate, host enters slow start mode when:
+  * For :ref:`NO_WAIT<envoy_v3_api_enum_value_config.cluster.v3.Cluster.CommonLbConfig.EndpointWarmingPolicy.NO_WAIT>` endpoint warming policy, immediately if it's cluster membership duration is within slow start window.
+  * For :ref:`WAIT_FOR_FIRST_PASSING_HC<envoy_v3_api_enum_value_config.cluster.v3.Cluster.CommonLbConfig.EndpointWarmingPolicy.WAIT_FOR_FIRST_PASSING_HC>` endpoint warming policy, upon passing an active healthcheck, given that it's cluster membership duration is within slow start window.
+
+Host exists slow start mode when:
+  * It leaves the cluster.
+  * It's cluster membership duration is greater than slow start window.
+  * For :ref:`WAIT_FOR_FIRST_PASSING_HC<envoy_v3_api_enum_value_config.cluster.v3.Cluster.CommonLbConfig.EndpointWarmingPolicy.WAIT_FOR_FIRST_PASSING_HC>` endpoint warming policy, if host's health status changes to anything rather than `Healthy`.
+
+Below is example of how requests would be distributed across hosts with Round Robin Loadbalancer, slow start window of 10 seconds, `NO_WAIT` ednpoint warming policy and 0.5 time bias.
```

Comment:
This is the first time you have discussed "time bias." Can you introduce this concept?